services/web/server/src/simcore_service_webserver/rest_handlers.py

```diff
@@ -68,3 +69,16 @@ async def get_config(request: web.Request):
     at runtime and the front-end can only get it upon request to /config
     """
     return web.json_response(data={"data": request.app[APP_SETTINGS_KEY].public_dict()})
+
+
+@routes.get(f"/{api_version_prefix}/scheduled_maintenance", name="get_scheduled_maintenance")
+async def get_scheduled_maintenance(request: web.Request):
+    """Check scheduled_maintenance table in redis"""
+
+    redis_client = get_redis_scheduled_maintenance_client(request.app)
+    hash_key = "maintenance"
+    # {"start": "2023-01-17T14:45:00.000Z", "end": "2023-01-17T23:00:00.000Z", "reason": "Release 1.0.4"}
+    maintenance_str = await redis_client.get(hash_key)
```

Comment:
I guess you need to validate the ``maintenance_str``
```python
MaintenanceData(BaseModel):
   start: datetime
   end: datetime
   reason: str

data = MaintenanceData.parse_raw(maintenance_str) 
```

Reply:
Feel free to validate it and to some tests in the next iteration 👍